commit-branch.md

```diff
@@ -176,30 +176,50 @@ No tak to už umíme, tak si pojďme ten soubor commitnout příkazem:
 
 ```
 $ git add --all
+```
+
+**GOTCHA:** defaultní editor pro úpravu commit messages a vůbec všeho textového v Gitu je VIM.
+A skoro nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat, takže nastavíme náš oblíbený editor - třeba Nano:
+```
+git config --global core.editor nano
+```
 
+A provedeme commit:
+```
 $ git commit
 ```
-**GOTCHA:** defaultní editor pro úpravu commit messages a vůbec všeho textového v Gitu je VIM.
-A nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat proto:
+
+Nyní napíšeme nějakou commit message - to, co jsme udělali. V našem případě asi něco jako:
+```
+přidání souboru pro poznámky notes.md
+```
+A uložíme
+
+```
+Ctrl + o
+<enter>
+```
+A ukončíme
 ```
-// zapisovací mód
-$ i
+Ctrl + q
 ```
+
+Pro adrenalinové zážitky ponecháme defaultním editorem VIM a provedeme commit:
```

Comment:
Tady sice píšeš, že tam necháme VIM, ale už sem nepíšeš příkaz. Takže bych buďto smazal tenhle řádek a pokračoval v Nano.

Nebo bych k tomuhle řádku https://github.com/webdev-js-evenings/git-workshop/pull/70/files#diff-44bc0bd3a3fcf64a688fd60caaaed01aR184 dopsal jenom poznámku, že to je pro fajnšmejkry.